test/unit/cleaner_test.rb

```diff
@@ -32,6 +32,27 @@ class TasksTest < ActiveSupport::TestCase
                   .find_execution_plans(filters: { 'uuid' => tasks_to_keep.map(&:external_id) }).size.must_equal tasks_to_keep.size
     end
 
+    describe "#orphaned_dynflow_tasks" do
+      # We can't use transactional tests because we're using Sequel for the cleanup query
+      self.use_transactional_tests = false
+      before do
+        skip "Sqlite is running testing Dynlfow DB in memory" if ActiveRecord::Base.connection.adapter_name == 'SQLite'
```

Comment:
We can't test the cleaner in sqite, because ActiveRecord and Sequel are running in separate DBs